components/molecule/autosuggest/src/index.js

```diff
@@ -288,9 +297,12 @@ MoleculeAutosuggest.propTypes = {
   /** list of values to be displayed on the select */
   options: PropTypes.array,
 
-  /** object generated w/ Reacte.createRef method to get a DOM reference of internal input */
+  /** object generated w/ Reacte.createRef method to get a DOM reference of wrapper div */
```

Comment:
/** object generated w/ React***e***.createRef method to get a DOM reference of wrapper div */